lib/TrackInfo.js

```diff
@@ -257,10 +257,44 @@ TrackInfo.expand = function(plain)
 	if (plain.mediaId)
 		//Set it
 		trackInfo.setMediaId(plain.mediaId);
-	//Gor each ssrc
-	for (const ssrc of plain.ssrcs || [])
-		//Add ssrc
-		trackInfo.addSSRC(ssrc);
+
+	//If we have an array of ssrcs
+	if (Array.isArray(plain.ssrcs))
+	{
+		//for each ssrc
+		for (const ssrc of plain.ssrcs)
+			//Add ssrc
+			trackInfo.addSSRC(ssrc);
+	} else {
+		//Check if we have named ssrcs
+		if ("media" in plain.ssrcs) 
+		{
+			//Get ssrc for media
+			const media = plain.ssrcs.media;
+			//Append to ssrc list
+			trackInfo.addSSRC(media);
+			//Check if we have named rtx ssrc
+			if ("rtx" in plain.ssrcs) 

```

Comment:
plays better with the type system :innocent: 
```suggestion
			if (plain.ssrcs.rtx) 
```

Reply:
rtx "could" be 0